protocol.html

```diff
@@ -267,7 +267,12 @@ <h2>Editor’s Draft, 2022-01-31</h2>
           <section id="abstract">
             <h2>Abstract</h2>
             <div datatype="rdf:HTML" property="schema:abstract">
-              <p>Solid Notifications Protocol.</p>
+              <p>
+                This specification defines a linked data-based protocol for subscribing to resources
+                in the Solid ecosystem. This protocol includes a mechanism for both discovering and
+                establishing subscriptions to linked data resources while also considering resource-based
```

Comment:
Is there any constraint such that the subscription of changes is limited to "linked data" (=RDF?) resources? 

```suggestion
                establishing subscriptions to resources while also considering resource-based
```